tests/test_norecipes_2.py

```diff
@@ -0,0 +1,116 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers._grouping_utils import IngredientGroup
+from recipe_scrapers.norecipes import NoRecipes
+from tests import ScraperTest
+
+
+class TestNoRecipesScraper(ScraperTest):
+
+    scraper_class = NoRecipes
+    test_file_name = "norecipes_2"
+
+    def test_host(self):
+        self.assertEqual("norecipes.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Marc", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual("Best Orange Chicken", self.harvester_class.title())
+
+    def test_category(self):
+        self.assertEqual("Entree", self.harvester_class.category())
+
+    def test_total_time(self):
+        self.assertEqual(30, self.harvester_class.total_time())
+
+    def test_yields(self):
+        self.assertEqual("4 servings", self.harvester_class.yields())
+
+    def test_image(self):
+        self.assertEqual(
+            "https://norecipes.com/wp-content/uploads/2017/06/orange-chicken-005.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ingredients(self):
+        expected_ingredients = [
+            "450 grams boneless skin-on chicken thighs ((cut into 1.5-inch pieces))",
```

Comment:
It's a nitpick, but could we remove the duplication of these brackets from the output ingredient text?

Reply:
Also interesting that they appear next to the ungrouped ingredients, and not in the grouped versions.  I guess there's some variance in the page data about those two representations..

Reply:
There was actually a larger issue with the way the data was being grabbed for the groups. I've resolved it now but the double brackets are what the test is requiring and what the schema is outputting. I could resolve it with...

```python
    def ingredients(self):
        ingredients_list = self.schema.ingredients()
        cleaned_ingredients = [ingredient.replace('((', '(').replace('))', ')') for ingredient in ingredients_list]
        return cleaned_ingredients

    def ingredient_groups(self):
        ingredients_list = self.ingredients()
        cleaned_ingredients = [ingredient.replace('((', '(').replace('))', ')') for ingredient in ingredients_list]
        return group_ingredients(
            cleaned_ingredients,
            self.soup,
            ".wprm-recipe-ingredient-group h4",
            ".wprm-recipe-ingredient-group li",
        )
```

I'll implement this now and if theres any opposition i can return it to the double parentheses.

Reply:
Looks great, thank you!